doc/reference/bluetooth/mesh/shell.rst

```diff
@@ -0,0 +1,733 @@
+
+
+``mesh period-set-unack <divisor>``
+-----------------------------------
+
+	Set the current Health Server publish period divisor. When a fault is detected, the Health Server will start publishing is fault status with a reduced interval. The reduced interval is determined by the Health Server publish period divisor: Fault publish period = Publish period / 2\ :sup:`divisor`.
+
+	* ``divisor``: The new Health Server publish period divisor.
+
+
+``mesh attention-get``
+----------------------
+
+	Get the current Health Server attention state.
+
+
+``mesh attention-set <timer>``
+------------------------------
+
+	Enable the Health Server attention state for some time.
+
+	* ``timer``: Duration of the attention state, in seconds (``0`` to ``255``)
+
+
+``mesh attention-set-unack <timer>``
+------------------------------------
+
+	Enable the Health Server attention state for some time without requesting a response.
+
+	* ``timer``: Duration of the attention state, in seconds (``0`` to ``255``)
+
+
+Heartbeat Server model
+======================
+
+``mesh add-fault <Fault ID>``
+-----------------------------
+
+	Register a new Fault for the Linux Foundation Company ID.
+
+	* ``Fault ID``: ID of the fault to register (``0x0001`` to ``0xFFFF``)
+
+
+``mesh del-fault [Fault ID]``
+-----------------------------
+
+	Remove registered faults for the Linux Foundation Company ID.
+
+	* ``Fault ID``: If present, the given fault ID will be deleted. If omitted, all registered faults will be cleared.
+
+
+Configuration database
+======================
+
+The Configuration database is an optional Mesh subsystem that can be enabled through the :option:`CONFIG_BT_MESH_CDB` configuration option. The Configuration database is only available on provisioner devices, and allows them to store all information about the mesh network. To avoid conflicts, there should only be one mesh node in the network with the Configuration database enabled. This node is the Configurator, and is responsible for adding new nodes to the network and configuring them.
+
+``mesh cdb-create [NetKey]``
+----------------------------
+
+	Create a Configuration database.
+
+	* ``NetKey``: Optional network key value of the primary network key (NetKeyIndex=0). Defaults to the default key value if omitted.
+
+
+``mesh cdb-clear``
+------------------
+
+	Clear all data from the Configuration database.
+
+
+``mesh cdb-show``
+-----------------
+
+	Show all data in the Configuration database.
+
+
+``mesh cdb-node-add <UUID> <addr> <num-elem> <NetKeyIdx> [DevKey]``
+-------------------------------------------------------------------
+
+	Manually add a mesh node to the configuration database. Note that devices provisioned with ``mesh provision`` and ``mesh provision-adv`` will be added automatically if the Configuration Database is enabled and created.
+
+	* ``UUID``: 128-bit hexadecimal UUID of the node. Any omitted bytes will be zero.
+	* ``addr``: Unicast address of the node.
```

Comment:
Maybe also mention that if addr = 0 the lowest available address will be used?

Reply:
Good catch, added.